explanation/intro-to/security.md

```diff
@@ -1,34 +1,32 @@
-immediate use, it is important to have a balanced understanding of your
-system's security posture based on how it will be used after deployment. It's
-important to take a layered approach so that your system's security is not
-dependent on a single point of defense.
+Security should always be considered when you install, deploy, and use an
+Ubuntu system. While a fresh installation of Ubuntu is usually safe for
+immediate use, your system's security posture will be very important depending
+on how it is used after it is deployed. You should always employ a layered
+approach to security so that that your system does not rely on a single
+point of defense.
 
 ## Server security guidance
 
-Since Ubuntu is so endlessly customizable, a full guide to security hardening
-is beyond the scope of this documentation.
-However, there are good practices and security-related packages that could be
-applied to almost any Ubuntu system. See our
-{ref}`security suggestions <security-suggestions>` page for an overview of the
-good habits and practices that can be adopted by anyone running an Ubuntu
-system to make it more secure. It's not necessary to apply every suggestion --
-and the list is not exhaustive by any means -- but each one used creates an
-extra layer of security.
-
-In a more advanced or complex setup, you may need to go further in your
-security outlook. There are specific packages available for your Server
-that will help with this, and we suggest some in the
-{ref}`advanced security <advanced-security>` section that you might want to
-consider for your use-case. Again, the list is not intended to be exhaustive,
-but rather a starting point.
-
-For a more thorough treatment of security in Ubuntu, we also recommend checking
-out the [Ubuntu Security documentation](https://ubuntu.com/security).
+Given Ubuntu has a wide range of customization options, this introduction cannot 
+provide a comprehensive security hardening guide. However, the security of nearly any 
+Ubuntu system can be enhanced by implementing certain best practices and security-related 
+packages. Our {ref}`security suggestions <security-suggestions>` page outlies those best 
+practices and packages. While you may not choose to implement every suggestion -- and 
+this list is not exhaustive -- each one can contribute an additional layer of security.
+
+If your Ubuntu setup is more advanced or more complex than others, you may 
+need to do more research into your security outlook. There are specific 
+packages available for Ubuntu Servers that can provide additional security, and
+we suggest some packages in the {ref}`advanced security <advanced-security>` 
+section that you might want to investigate. Again, the list in this section
+is not  intended to be exhaustive, but rather a starting point.
+
+To read the most thorough treatment of security in Ubuntu, go to the
+[Ubuntu Security documentation](https://ubuntu.com/security).
```

Comment:
```suggestion
For the most thorough treatment of security in Ubuntu, we also recommend reading the [Ubuntu Security documentation](https://ubuntu.com/security).
```

Reply:
I missed this the first time around, the original intention of the pointer to the security docs was to provide it as an additional suggestion, but the wording change implies that the user should only look at the security docs (which is not the case, since our security docs are more specific to Server users). I've updated the original wording to incorporate the spirit of your change while also retaining the original intent.